app/views/checkouts/checkout.html.slim

```diff
@@ -0,0 +1,74 @@
+section.purchase
+  h1.text-center Pagamento
+  - if session[:shopping_cart].any?
+    .container
+      .row
+        .col-md-10.col-centered
+          .purchase-table
+            .purchase-info
+              - total = 0
+              - session[:shopping_cart].each do |item|
+                - offer = Offer.find(item['id'])
+                - total += offer.value*item['quantity']
+                .offer-info__products
+                  .row
+                    .col-md-12
+                      strong = offer.title
+                  .row
+                    .col-md-4
+                      span.btn.btn--md Unidade
+                    .col-md-4
+                      span.btn.btn--md Quantidade
+                    .col-md-4
+                      span.btn.btn--md Total
+                  .row
+                    .col-md-4
+                      span.btn.btn--md = number_to_currency offer.value
+                    .col-md-4.text-center
+                      span.btn.btn--md = item['quantity']
+                    .col-md-4
+                      span.btn.btn--md = number_to_currency offer.value*item['quantity']
+
+            .purchase-form
+              = form_tag process_payment_path, id: 'js-payment-form' do
+                = hidden_field_tag :token, nil, readonly: true, size: 64
+
+                h3 1. Selecione a opção de pagamento
+                .btn-group.payment-method-btn-group data-toggle="buttons"
+                  label#js-credit-card-checkbox-label.btn.btn-default.active Cartão de crédito
+                  = check_box_tag :method, '', true, id: 'js-payment-method-credit-card', style: 'display: none;'
```

Comment:
Basically because for Iugu when you pass a method value this must be `bank_slip`, and when you pass a token this must be `credit_card`.